_content/av/back-to-nature_yuttadhamo.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Back to nature"
+authors:
+- yuttadhammo
+external_url: "https://tunein.com/podcasts/Religion--Spirituality-Podcas/Dhamma-Talks-(Part-3)-p2926505/?topicId=220379444"
+source_url: ""
```

Comment:
These Yuttadhammo mp3s all come from https://static.sirimangalo.org/diraudio/Yuttadhammo/
This particular file is:
```yaml
source_url: "https://static.sirimangalo.org/diraudio/Yuttadhammo/Daily/130401_nature.mp3"
```